### PR TITLE
cmr-6631 - adding sequence and uniq constraint

### PR DESCRIPTION
**Pull request by jceaser**
Table was created in a previous pull request (same ticket number), adding a uniq constraint and a sequence for the primary key

### CODE REVIEW

File: metadata-db-app/src/cmr/metadata_db/migrations/072_add_cmr_sub_notifications_table.clj [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private sub-notifications-column-sql
+ "id NUMBER,
+ subscription_concept_id VARCHAR(255) NOT NULL,
+ last_notified_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL")
+
+ (def ^:private sub-notifications-constraint-sql
+   (str "CONSTRAINT sub_notifications_pk PRIMARY KEY (id),"
+   "CONSTRAINT subscriptions_nid_rid UNIQUE (subscription_concept_id)"))
```
Comment (ygliuvt) [inline metadata-db-app/src/cmr/metadata_db/migrations/072_add_cmr_sub_notifications_table.clj:13]: This constraint name needs to be changed to something like `sub_cid`.

File: metadata-db-app/src/cmr/metadata_db/migrations/072_add_cmr_sub_notifications_table.clj [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+(defn- create-notification-sequence
+ "Create Sequences used by this table"
+  []
+  (h/sql "CREATE SEQUENCE cmr_sub_notifications_seq"))
+
+(defn up
+  "Migrates the database up to version 72."
+  []
+  (println "cmr.metadata-db.migrations.072-add-cmr-sub-notifications-table up...")
+  (create-notification-table)
```
Comment (ygliuvt) [inline metadata-db-app/src/cmr/metadata_db/migrations/072_add_cmr_sub_notifications_table.clj:31]: nit: Just a comment. You don't have to make changes. I am really not a fan with these helper functions. Just write the whole thing out without any functions will be so much more readable.

**Review by ygliuvt — APPROVED** [review]

**Comment by ygliuvt** [comment]
Need to resolve merge conflict.

**Review by mschmele — APPROVED** [review]
Seconding Leo's comment about merge conflicts

File: metadata-db-app/src/cmr/metadata_db/migrations/072_add_cmr_sub_notifications_table.clj (hunk 2 above)
Comment (jceaser) [reply metadata-db-app/src/cmr/metadata_db/migrations/072_add_cmr_sub_notifications_table.clj:31]: that is good to know. I got the idea from other migrations and I don't know what is expected.

**Pull request merged**